### PR TITLE
Rename Ruckus Unleashed to Ruckus, update docs

### PR DESCRIPTION
**Pull request by ms264556**
## Proposed change

Changed integration name from "Ruckus Unleashed" to "Ruckus": this integration now supports all Ruckus controller platforms.

Also update instructions to cover extra platforms and different connection requirements (the integration now connects over HTTPS instead of SSH).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125392
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Enhanced clarity in the documentation for Ruckus Networks integration with Home Assistant.
	- Added support details for multiple Ruckus access point types.

- **Documentation**
	- Updated title and description for better specificity.
	- Improved readability and formatting of the integration guide.
	- Revised configuration instructions for Ruckus One connection.
	- Expanded troubleshooting section with new accessibility details.

### CODE REVIEW

File: source/_integrations/ruckus_unleashed.markdown [hunk 1]
```diff
@@ -16,38 +16,52 @@ ha_platforms:
...
 
 ## Configuration
 
-To add a Ruckus Unleashed device to your installation, go to **Settings** -> **Devices & services**, click the `+` button, then select **Ruckus** from the list of integrations.
+To add a Ruckus Unleashed device to your installation, go to **Settings** -> **Devices & services**,
+click the `+` button, then select **Ruckus** from the list of integrations.
 
-It is required to configure the IP address of your **master access point**. See the section Access Points on the management web interface. And perhaps consider to set a preferred master (Admin & Services>System>System Info>Preferred master).
+Enter be the **IP address / hostname**, **Username** and **Password** which you use to connect
+to your Ruckus controller's web dashboard.
```
Comment (c0ffeeca7) [inline source/_integrations/ruckus_unleashed.markdown]: I can't make a suggestion over deleted lines. But I think this section could be replaced by a prerequisites section, followed by the standard config flow include:

## Prerequisites

- **IP address / hostname**, **Username** and **Password** which you use to connect
to your Ruckus controller's web dashboard.

{% include integrations/config_flow.md %}

File: source/_integrations/ruckus_unleashed.markdown [hunk 2]
```diff
@@ -16,38 +16,52 @@ ha_platforms:
...
 
-You will have to create a user on the device which is a **Monitoring Admin**. Login to the Ruckus Unleashed admin UI and follow these steps:
+### Ruckus Unleashed
 
- - [Create a new role](https://docs.ruckuswireless.com/unleashed/200.1.9.12/t-ConfigUserRoles.html).
-   - Check **Allow Unleashed Administration**.
-   - Select the **Monitoring Admin (Monitoring and viewing operation status only)** radio button. 
- - [Create a new user](https://docs.ruckuswireless.com/unleashed/200.1.9.12/t-AddingNewUsersInternal.html) with the new role.
+You may enter the **IP address / hostname** of any access point as the Host.
+If you've configured an Unleashed Management Interface then use this instead.
```
Comment (c0ffeeca7) [inline source/_integrations/ruckus_unleashed.markdown]:
```suggestion
If you've configured an Unleashed Management Interface, then use this instead.
```

File: source/_integrations/ruckus_unleashed.markdown [hunk 3]
```diff
@@ -16,38 +16,52 @@ ha_platforms:
...
 
-## Troubleshooting
+### Ruckus One
+
+You'll need to use your Ruckus One dashboard to create an Application Token. Go to the bottom of the
+**Administration** > **Settings** screen and click the **Add Token** link. Choose any
```
Comment (c0ffeeca7) [inline source/_integrations/ruckus_unleashed.markdown]:
```suggestion
**Administration** > **Settings** screen and select the **Add Token** link. Choose any
```

File: source/_integrations/ruckus_unleashed.markdown [hunk 4]
```diff
@@ -16,38 +16,52 @@ ha_platforms:
...
 
-Terminal:
+## Limitations
 
-```bash
-ssh <ruckus_ip>
+This integration is not currently suitable for large multi-venue SmartZone or Ruckus One networks: there
+is no way to filter devices by Venue or Zone.
+
+If you've configured your access points with an extended Client Inactivity Timeout then this is how long
```
Comment (c0ffeeca7) [inline source/_integrations/ruckus_unleashed.markdown]:
```suggestion
If you've configured your access points with an extended Client Inactivity Timeout, then this is how long
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/ruckus_unleashed.markdown (hunk 1 above)
Comment (ms264556) [reply source/_integrations/ruckus_unleashed.markdown]: Yes, this is much better thanks!

File: source/_integrations/ruckus_unleashed.markdown [hunk 5]
```diff
@@ -16,38 +16,51 @@ ha_platforms:
...
 
-## Configuration
+## Prerequisites
 
-To add a Ruckus Unleashed device to your installation, go to **Settings** -> **Devices & services**, click the `+` button, then select **Ruckus** from the list of integrations.
+- **IP address / hostname**, **Username** and **Password** which you use to connect
+    to your Ruckus controller's web dashboard.
 
-It is required to configure the IP address of your **master access point**. See the section Access Points on the management web interface. And perhaps consider to set a preferred master (Admin & Services>System>System Info>Preferred master).
+{% include integrations/config_flow.md %}
```
Comment (c0ffeeca7) [inline source/_integrations/ruckus_unleashed.markdown]:
```suggestion
```

File: source/_integrations/ruckus_unleashed.markdown [hunk 6]
```diff
@@ -16,38 +16,51 @@ ha_platforms:
...
+### Ruckus One
+
+You'll need to use your Ruckus One dashboard to create an Application Token. Go to the bottom of the
+**Administration** > **Settings** screen and select the **Add Token** link. Choose any
+**Application Name**, for example `Home Assistant`. The **Token Scope** can be **Read Only**.
 
-For this platform to work, the Ruckus Unleashed device will need to be accessible over SSH. If you are having trouble with Home Assistant not connecting, make sure the user you created above can log in to SSH and can run privileged commands.
+When Home Assistant prompts for Ruckus connection details, use the full URL of a Ruckus One
+dashboard page as the Host (such as `https://asia.ruckus.cloud/5dd1000334cc2a01fcf28a740a6c95cf/t/dashboard`),
+your Token **Client ID** as the Username & your Token **Shared Secret** as the Password.
```
Comment (c0ffeeca7) [inline source/_integrations/ruckus_unleashed.markdown:52]:
```suggestion

{% include integrations/config_flow.md %}

```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @ms264556 :+1:

**Pull request merged**